sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/handler/ReceiverUnsettledDeliveries.java

```diff
@@ -56,15 +56,17 @@ public final class ReceiverUnsettledDeliveries implements AutoCloseable {
     // (TODO: anuchan; consider parametrizing the value of deliveryTag?).
```

Comment:
When the type was originally designed, we were calling it `DeliveryHandler`, then there was feedback during the offline review that, per AMQP protocol, the concept of delivery handling exists for both Receiver and Sender. Though we don’t have Sender delivery handling in azure-core-amqp now, at that time we thought it’s good to prefix with Receiver to avoid confusion.

Reply:
But I don’t have a specific preference, I’m fine with dropping ‘Receiver’ prefix or keeping it. Pls let me know your thoughts.